test/handler.test.js

```diff
@@ -628,6 +628,30 @@ describe('strong-error-handler', function() {
         .expect('Content-Type', /^application\/json/, done);
     });
 
+    it('dishonors accepted content-type if mandated by contentType option',
+      function(done) {
+        givenErrorHandlerForError(new Error('Some error'), {
+          contentType: 'application/json',
+        });
+        request.get('/')
+          // `application/json` will be used because its provided first
+          .set('Accept', 'text/html')
+          .expect('Content-Type', /^application\/json/, done);
+      }
+    );
+
+    it('dishonors default content-type if mandated by contentType option',
+      function(done) {
+        givenErrorHandlerForError(new Error('Some error'), {
+          defaultType: 'application/json',
+          contentType: 'text/html',
+        });
+        request.get('/')
+          // `application/json` will be used because its provided first
```

Comment:
> `application/json` will be used because its provided first

I find this rather confusing for our users. IIUC, when the user configures both `defaultType` and `contentType`, then the current implementation will use `defaultType` and ignore `contentType`, is that correct?

This behavior should be described in the README.